magento/magento2
# Will replace images during CSV import

### Description
Built on top of https://github.com/magento/magento2/pull/21146

This PR adds image deletion so after import product will have images specified in the CSV. New images will be added, existing will not be duplicated and images not mentioned in the CSV will be removed 

### Fixed Issues
magento/magento2#14398: Images Not Replaced via CSV Import

### Manual testing scenarios (*)

1. Import product CSV using Add/Update method multiple times
2. Check if image duplication happens or not

There's many testing scenario's actually. Some of them being:
- Use HTTP references in images
- Same filename different file
- Different filename same file
- Same file path for multiple products
- Change image labels/titles, ensure they're updated
- Manipulate thumbnail, small image and base image tags, and ensure it's updated properly after import
- etc.
